cpp/src/column/column.cu

```diff
@@ -275,6 +278,123 @@ struct create_column_from_view_vector {
 
 };
 
+template <typename T>
+struct partition_map_fn {
```

Comment:
This was the lambda that was causing problems with `thrust::tabulate`

Reply:
I've never been able to successfully isolate a reproducer for this issue. If you have the time, try it out and see if you can get the issue to reproduce in isolation so we can file a bug.